lib/services/auth/service.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:liga_inggris_mobile/app/core/base/base_service.dart';
+import 'package:liga_inggris_mobile/app/core/handler/service_result.dart';
+import 'package:liga_inggris_mobile/services/auth/model.dart';
+import 'package:liga_inggris_mobile/services/club/model.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class AuthService extends BaseService {
+  final List<UserModel> userList = [
```

Comment:
karna ini cuma dipake di class ini, jadiin private variable
```suggestion
  final List<UserModel> _userList = [
```